### PR TITLE
[PE-9] Display hyperlink to HHS disclosure on RADx footer

### PR DESCRIPTION
**Pull request by kyuksel**
https://broadworkbench.atlassian.net/browse/PE-9

## Before
image

## After
image

### CODE REVIEW

File: src/components/FooterWrapper.js [hunk 1]
```diff
@@ -76,6 +76,10 @@ const FooterWrapper = ({ children, alwaysShow, fixedHeight }) => {
     a({ href: Nav.getLink('privacy'), style: styles.item }, 'Privacy Policy'),
     div({ style: styles.item }, '|'),
     a({ href: Nav.getLink('terms-of-service'), style: styles.item }, 'Terms of Service'),
+    isRadX() && popoutItem({
```
Comment (nawatts) [inline src/components/FooterWrapper.js]: Would it make sense to put this after "Security"? This seems like a more specific part of the general "Security" topic.

**Review by nawatts — APPROVED** [review]

File: src/components/FooterWrapper.js (hunk 1 above)
Comment (cahrens) [reply src/components/FooterWrapper.js]: I agree that swapping the order would make more sense.

**Review by cahrens — APPROVED** [review]
But may wish to swap the order.

File: src/components/FooterWrapper.js (hunk 1 above)
Comment (kyuksel) [reply src/components/FooterWrapper.js]: Good thinking. Changed in [1fc8c16](https://github.com/DataBiosphere/terra-ui/pull/3379/commits/1fc8c16df1831c2997de93e5656e79a97d3394d8).

**Pull request merged**